src/CorePolicyRamCloud.h

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2009-2016 Stanford University
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR(S) DISCLAIM ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL AUTHORS BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#ifndef RAMCLOUD_COREPOLICYRAMCLOUD_H
+#define RAMCLOUD_COREPOLICYRAMCLOUD_H
+
+#include "Arachne/CorePolicy.h"
+
+#define BASE_THREAD_CLASS 0
+#define DISPATCH_THREAD_CLASS 1
+#define DISPATCH_HT_THREAD_CLASS 2
+
+extern CorePolicy* corePolicyRamCloud;
+
+class CorePolicyRamCloud : public CorePolicy {
```

Comment:
I think I see why you named this class the way you did (better collation?), and I see the advantage of that, but unfortunately, that's not our coding style. Our style is to put the qualifier first, e.g., BasicTransport, or RamCloudCorePolicy.